src/modules/users/components/TokenActivation/TokenActivationContent/StakesTab.tsx

```diff
@@ -4,10 +4,17 @@ import { FormattedMessage, defineMessages } from 'react-intl';
 import { SpinnerLoader } from '~core/Preloaders';
 import { Address } from '~types/index';
 import { useClaimableStakedMotionsQuery } from '~data/generated';
+import Heading from '~core/Heading';
+
+import ClaimAllButton from './ClaimAllButton';
 
 import styles from './TokenActivationContent.css';
 
 const MSG = defineMessages({
+  tabTitle: {
+    id: 'users.TokenActivation.TokenActivationContent.ClaimsTab.noClaims',
```

Comment:
.ClaimsTab..... - but was renamed to StakesTab. I can update when i make my changes for the dialog.